src/hotspot/share/gc/z/zMetronome.cpp

```diff
@@ -28,7 +28,7 @@
 #include "utilities/ticks.hpp"
 
 ZMetronome::ZMetronome(uint64_t hz) :
-    _monitor(Monitor::leaf, "ZMetronome", false, Monitor::_safepoint_check_never),
+    _monitor(Monitor::leaf, "ZMetronome", Monitor::_safepoint_check_never),
```

Comment:
Again the "false" to "true" stands out here, but again I have to assume this is not a lock that is used by JavaThreads. Makes me wonder whether these locks can never encounter contention by the VMThread (otherwise they would assert).

Reply:
Also assume that this is only taken by GC threads.